client/catalog/src/app/repl/Repl.svelte

```diff
@@ -0,0 +1,80 @@
+<script>
+import client from '../../data/client.js';
+import { breadcrumbStore } from '../stores.js';
+import { onMount } from 'svelte';
+import HLRowButton from '../../ui/HLRowButton.svelte';
+import HLTable from '../../ui/HLTable.svelte';
+import HLButton from '../../ui/HLButton.svelte';
+import HLRow from '../../ui/HLRow.svelte';
+import HLTextBig from '../../ui/HLTextBig.svelte';
+import HLSelect from '../../ui/HLSelect.svelte';
+import CodeMirror from 'codemirror';
+import 'codemirror/mode/python/python.js';
+import 'codemirror/addon/selection/active-line.js';
+
+breadcrumbStore.set(
+	[{
+		href: "/repl",
+		text: "Repl",
+	}]
+);
+
+var cm;
+var repl;
+
+onMount(async () => {
+	cm = CodeMirror.fromTextArea(document.querySelector('#code'), {
+		mode: {name:"python"},
+		lineNumbers: true,
+		indentUnit: 4,
+		theme: "duotone-dark",
+	});
+	cm.focus();
+	repl = CodeMirror.fromTextArea(document.querySelector('#repl'), {
```

Comment:
also since you've now hardcoded the them and settings in 3 places, it's time to pull this out into a ui component "HLCodeMIrror" or something